ovos_dinkum_listener/voice_loop/voice_loop.py

```diff
@@ -443,7 +522,11 @@ def _before_cmd(self, chunk):
                 # Reset
                 self.speech_seconds_left = self.speech_seconds
 
-    def _in_cmd(self, chunk):
+    def _in_cmd(self, chunk: bytes):
+        """
+        Handle audio chunks identified as speech by VAD
```

Comment:
this is after WakeWord step, not necessarily all chunks are identified as speech by VAD, in this case VAD is being used to detect when speech ends, some silence chunks will feed to this method in between words or until the threshold for max silence is reached

Reply:
Doesn't this call `before_cmd` until speech is detected and then `after_cmd` after speech is ended?

Reply:
if there is no speech before_cmd it still times out and sends silence here potentially, and we get silence between words and at end of sentence feed to this method, the docstrs seem to ensure all audio chunks were tagged as speech but they might not have been classified as such by VAD. after_cmd is called after a min amount of silence some of which might also be sent to this handler